openapi/resource-server.yaml

```diff
@@ -1012,6 +1020,100 @@ components:
+      type: object
+      examples:
+        - id: 'https://ilp.rafiki.money/outgoing-payments/8c68d3cc-0a0f-4216-98b4-4fa44a6c88cf'
+          walletAddress: 'https://ilp.rafiki.money/alice/'
+          failed: false
+          receiver: 'https://ilp.rafiki.money/aplusvideo/incoming-payments/45d495ad-b763-4882-88d7-aa14d261686e'
+          receiveAmount:
+            value: '2500'
+            assetCode: USD
+            assetScale: 2
+          debitAmount:
+            value: '2600'
+            assetCode: USD
+            assetScale: 2
+          sentAmount:
+            value: '2500'
+            assetCode: USD
+            assetScale: 2
+          grantSpentDebitAmount:
+            value: '2600'
+            assetCode: USD
+            assetScale: 2
+          grantSpentReceiveAmount:
+            value: '2500'
+            assetCode: USD
+            assetScale: 2
+          createdAt: '2022-03-12T23:20:50.52Z'
+          updatedAt: '2022-04-01T10:24:36.11Z'
+          metadata:
+            description: APlusVideo subscription
+            externalRef: 'customer: 847458475'
+      properties:
+        id:
+          type: string
+          format: uri
+          description: The URL identifying the outgoing payment.
+          readOnly: true
+        walletAddress:
+          type: string
+          format: uri
+          description: The URL of the wallet address from which this payment is sent.
+          readOnly: true
+        quoteId:
+          type: string
+          format: uri
+          description: The URL of the quote defining this payment's amounts.
+          readOnly: true
+        failed:
+          type: boolean
+          description: Describes whether the payment failed to send its full amount.
+          default: false
+        receiver:
+          $ref: ./schemas.yaml#/components/schemas/receiver
+          description: The URL of the incoming payment that is being paid.
+        receiveAmount:
+          $ref: ./schemas.yaml#/components/schemas/amount
+          description: The total amount that should be received by the receiver when this outgoing payment has been paid.
+        debitAmount:
+          $ref: ./schemas.yaml#/components/schemas/amount
+          description: The total amount that should be deducted from the sender's account when this outgoing payment has been paid.
+        sentAmount:
+          $ref: ./schemas.yaml#/components/schemas/amount
+          description: The total amount that has been sent under this outgoing payment.
+        grantSpentDebitAmount:
+          $ref: './schemas.yaml#/components/schemas/amount'
+          description: The total amount already deducted from the sender's account using the current outgoing payment grant.
+        grantSpentReceiveAmount:
+          $ref: './schemas.yaml#/components/schemas/amount'
+          description: The total amount already received (by all receivers) against the current outgoing payment grant.
```

Comment:
Thinking about this, we should use either "against" or "using" in both of the descriptions.
Also, we should add something that says these amounts are "successfully received" or "successfully deducted", to clear up the fact that these amounts do _not_ include this newly created outgoing payment (since we've just created it, and it hasn't actually sent funds yet)

Reply:
Changed to:

> description: The total amount successfully deducted from the sender's account using the current outgoing payment grant.

and:

> description: The total amount successfully received (by all receivers) using the current outgoing payment grant.